lib/github_service/commands/add_label.rb

```diff
@@ -1,20 +1,21 @@
+require 'did_you_mean'
+
 module GithubService
   module Commands
     class AddLabel < Base
       private
 
       def _execute(issuer:, value:)
         valid, invalid = extract_label_names(value)
+        process_extracted_labels(valid, invalid)
 
         if invalid.any?
-          message = "@#{issuer} Cannot apply the following label#{"s" if invalid.length > 1} because they are not recognized: "
-          message << invalid.join(", ")
-          issue.add_comment(message)
+          issue.add_comment(invalid_label_message(issuer, invalid))
         end
 
         if valid.any?
           valid.reject! { |l| issue.applied_label?(l) }
-          issue.add_labels(valid)
+          issue.add_labels(valid) unless valid.empty?
```

Comment:
For readability, can we use `if valid.any?` as opposed to the double negative?

Reply:
Sure, that works.  👍